webapi/src/main/scala/org/knora/webapi/slice/ontology/api/OntologiesEndpoints.scala

```diff
@@ -100,6 +163,15 @@ final case class OntologiesEndpoints(baseEndpoints: BaseEndpoints) {
 
   val endpoints =
     Seq(
+      putOntologiesCardinalities,
+      postOntologiesCandeletecardinalities,
+      patchOntologiesCardinalities,
+      putOntologiesGuiorder,
+      getOntologiesClassesIris,
+      getOntologiesCandeleteclass,
+      deleteOntologiesClasses,
+      deleteOntologiesComment,
+      postOntologiesProperties,
```

Comment:
Would it make sense to sort entire list alphabetically?

Reply:
I have kept the original ordering, see also my other comment.